website/app/components/Head.tsx

```diff
@@ -0,0 +1,39 @@
+import { Helmet } from 'react-helmet';
+import { DocumentationLoader } from '~/loaders/documentation.server';
+import codeHikeStyles from '@code-hike/mdx/dist/index.css';
+
+export const Head = ({ data }: { data: DocumentationLoader }) => {
+  const favicon = getFavicon({ data });
+
+  return (
+    //@ts-ignore
+    <Helmet>
+      <link rel="icon" href={favicon} />
+      {data.config.experimentalMath && (
+        <link
+          rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/katex@0.15.0/dist/katex.min.css"
+        />
+      )}
+      {data.config.experimentalCodehike && <link rel="stylesheet" href={codeHikeStyles} />}
```

Comment:
I think this needs a `key` if the page changes, won't this stay there because it doesn't know which to remove?